model/maslout/gen/system.mark

```diff
@@ -1,5 +1,5 @@
 .invoke MarkSystemConfigurationPackage( "sysmaslout" )
 .invoke TagDynamicMemoryAllocationOn( 10 )
 .invoke TagInstanceLoading()
-.invoke TagMaximumStringLength( 840 )
+.invoke TagMaximumStringLength( 65536 )
```

Comment:
wasn't max string length tagged to 180000 earlier in masl or m2x?  Should they be consistent?